conf/routes

```diff
@@ -12,3 +12,4 @@ GET     /assets/*file               controllers.Assets.versioned(path="/public",
 
 # API endpoints
 GET     /pass-bakery/status         controllers.APIController.getStatus()
+GET     /pass-bakery/db             controllers.APIController.getBakeryProducts()
```

Comment:
lets be more specific on this endpoint naming

Reply:
Endpoint renamed to `/pass-bakery/db-tables`